backend/package.json

```diff
@@ -14,6 +14,11 @@
     "cors": "^2.8.5",
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
+    "express-mongo-sanitize": "^2.0.2",
+    "express-rate-limit": "^5.2.6",
+    "helmet": "^4.4.1",
+    "mongoose": "^5.11.15",
```

Comment:
Hey, there's an error still. In the package.json file, line 20 and 23 are same + there is to be a comma at the end of line 21
Before issuing a PR next time, do run the app locally as these are some fatal bugs that totally break the server from even starting.

Reply:
I am sorry. I will check that in mind. I just made the changes though.